_bootstrap.py

```diff
@@ -0,0 +1,51 @@
+# _bootstrap.py
+
+# *** DON'T RUN THIS DIRECTLY ***
+# Run symlink_bootstrap.py to link this bootstap to working dirs (dirs with scripts ran directly)
+
+# handle path issues so scripts run in subfolders can see config files and util scripts,
+# as well as adding other python interpreters to path instead of needing to do it repeatedly.
+
+# *** be sure to import _bootstrap BEFORE any relative imports ***
+
+# e.g. inside of `1_data_prep/1_get_mission_altitude_driver_script.py`:
+# import _bootstrap
+# from configs.path_config import path_config
+
+import os
+import sys
+from pathlib import Path
+
+# ----- Resolve relative imports (e.g. `config/`) -----
+
+here = Path(__file__).parent.resolve() # current dir
+project_rt = here.parent # project root
+
+# ensure root is on sys.path
+root_str = str(project_rt)
+if root_str not in sys.path:
+    sys.path.insert(0, root_str)
+
+# add every first‑level folder under root
+for sub in project_rt.iterdir():
+    if sub.is_dir() and not sub.name.startswith('.'):
+        sub_str = str(sub.resolve())
+        if sub_str not in sys.path:
+            sys.path.insert(0, sub_str)
+
+
+# ----- Add alt Python interpreter dirs to path (e.g. Automate Metashape) -----
+try:
+    from configs.path_config import path_config
+except ImportError:
+    raise FileNotFoundError("ERROR: Could not find path config to add interpreters to system path")
+
+# ADD OTHER INTERPRETES HERE AS NEEDED
+INTERPRETER_PATHS = [
```

Comment:
This isn't actually an interpreter but rather it's a folder within `automate-metashape` called [python](https://github.com/open-forest-observatory/automate-metashape/tree/main/python).

Reply:
It has a driver script within it that we need to call, so maybe `SCRIPTS_PATHS`?